app/src/test/java/com/team7/cmput301/android/theirisproject/RecordTest.java

```diff
@@ -62,20 +66,15 @@ public void testEditGeoLocation() {
     public void testEditBodyLocation() {
         Record record = getTestRecord();
 
-        BodyLocation updatedBodyLocation = null;
+        ArrayList<BodyLocation> updatedBodyLocation = new ArrayList<>();
```

Comment:
I know this is a test but definitely try to use the interface type as the reference if you can (List<T>)